docs/user-documentation/searching.md

```diff
@@ -0,0 +1,60 @@
+
+#### Edit Tab
+
+The 'Edit' tab allows repository managers to configure how the index works as a whole, including its name, the data sources—entity types—it can index (including which specific content types or taxonomies will be indexed), which server it is connected to, and other SOLR-specific options.
+
+'Content' (types) is the only data source enabled by default. Selecting 'Taxonomy term' will enable searching taxonomies which is recommended if the repository uses taxonomies for subjects or other discovery points. Once the data sources are enabled a configuration box for each of them will appear in a section just below the list of data sources. This allows repository managers to select which content types (or taxonomy vocabularies) will be included in the index. By default all the content types, and vocabularies if the taxonomy data source is enabled, are indexed.
+
+**Note:** the defaults assume a repository is adding content using the web interface. If a repository manager plans on bulk-loading content they should disable the 'Index items immediately' option in the expandable 'Index Options' box and increase the 'Cron batch size' option.
+
+#### Fields Tab
+
+The 'Fields' tab allows repository managers to select which fields will be indexed. The default set of fields enabled come from a standard Drupal installation and do not reflect the fields Islandora adds for 'Repository Item'. **Repository managers need to add the fields necessary for their Islandora instance.**
+
+To add a field, click the '+ Add fields' button. A shadow-box will appear with a list of the fields available for the index.
+
+Some fields, such as the 'Body' ('body') field provided by Drupal, have multiple subfields which can be completely different values or variations on the same value. Click on the plus-sign next to the field to show the available subfields. In most cases repository managers can click the 'Add' button by the field and ignore the subfields as it will index the field as it is rendered to users. Only select the subfields if you understand the field's type, the value returned by the subfield, and how it will impact user searching. Similar to the complex fields, entity reference fields, such as Tags ('field_tags'), allow you to select fields from the referenced entity to index.
+
+Once the fields are added they can be configured further on the 'Fields' tab, although the label, machine name, and type usually don't need to be changed. The 'Type' setting has several different types of Full-text processing available, which may be of interest. Each is described in the expandable 'Data Types' box at the bottom of the page. The 'Boost' setting allows repository managers to increase the weight of particular fields when calculating search relevancy.
```

Comment:
Change: The 'Type' **dropdown** has several different **options** of Full-text processing available which may be of interest.

Reply:
It would be positive to speak about field properties and not subfields. There is not such a think of a subfield in drupal 8. Fields can have many properties, some exposed and indexable other not (search api limitations) and even the default, visible and renderable value is a property, the a default one (->value).

Reply:
@DiegoPino, right. I was using my internal conceptualization of Fields (when I was trying to figure out how they worked) rather than the Drupal terminology. I'll fix it.

Reply:
@DiegoPino, I've updated this part. Let me know what you think.